workshops/models.py

```diff
@@ -272,9 +272,10 @@ class Event(models.Model):
     '''Represent a single event.'''
 
     published  = models.BooleanField(default=False)
-    site       = models.ForeignKey(Site)
+    site       = models.ForeignKey(Site, null=True, on_delete=models.SET_NULL)
```

Comment:
Same comments as for `Person.airport`.  I don't want site-less `Event`s in the database, and I don't want to blow away associated `Event`s when someone tries to remove a `Site`.